modules/oe_content_project/config/install/field.field.node.oe_project.oe_project_locations.yml

```diff
@@ -12,7 +12,7 @@ entity_type: node
 bundle: oe_project
 label: 'Project locations'
 description: ''
-required: false
+required: true
```

Comment:
Looks like it is fine to leave this field as required (declared in specification) even if we have an issue with the widget of location module.